amf-cli/shared/src/test/resources/validations/raml/nested-json-schema-ref/api.raml

```diff
@@ -0,0 +1,6 @@
+#%RAML 1.0
+title: sMaximoLegacy
```

Comment:
Please change API and related schema contents

Reply:
true, done 👌🏽